### ISSUE TITLE
Descriptions for --no-* flags should indicate they disable the associated feature

### ISSUE THREAD

**Issue by mark-vieira** [gradle/gradle#3488]
### Expected Behavior

When running `gradle --help` the descriptions for the `--no-*` options should explicitly state that they _disable_ the corresponding feature.

### Current Behavior

Currently the descriptions for the `--no-*` options when running `--help` are simply a copy of their "enable" counterpart. This is a bit confusing it's not clear that these options actually _disable_ these features. In some cases the description actually includes the work "enable".

```
--no-build-cache          Enables the Gradle build cache. Gradle will try to reuse outputs from previous builds. [incubating]
--no-configure-on-demand  Configure necessary projects only. Gradle will attempt to reduce configuration time for large multi-project builds. [incubating]
--no-daemon               Uses the Gradle Daemon to run the build. Starts the Daemon if not running.
--no-parallel             Build projects in parallel. Gradle will attempt to determine the optimal number of executor threads to use. [incubating]
--no-scan                 Creates a build scan. Gradle will emit a warning if the build scan plugin has not been applied. (https://gradle.com/build-scans) [incubating]
```

### Steps to Reproduce (for bugs)

Run `gradle --help`.

**Comment by big-guy** [comment]
This is fixed in 4.4: https://github.com/gradle/gradle/issues/3232

**Comment by mark-vieira** [comment]
Should have done a search first. Thanks, Sterling.
